dts/bindings/misc/intel,timeaware-gpio.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel TGPIO node
```

Comment:
TAGPIO, if you choose tagpio to be the short name

Reply:
Moved from TAGPIO to TGPIO all over the PR.